api/room/schema_query.py

```diff
@@ -173,7 +196,7 @@ def resolve_all_remote_rooms(self, info):
             calendar_list = service.calendarList().list(
                 pageToken=page_token).execute()
             for room_object in calendar_list['items']:
-                if 'andela.com' in room_object['id'] and room_object['id'].endswith( # noqa
+                if 'andela.com' in room_object['id'] and room_object['id'].endswith(  # noqa
                         'resource.calendar.google.com'):
```

Comment:
Please put the noqa outside the brackets

Reply:
@kampkelly this has been worked on.